build

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2.7
```

Comment:
That sounds good to me. Let's drop the .7 from this request, and per my
other comment, if we can add documentation to this effect, that would be
best for now.

How would you feel about putting this into a new file called, say,
RECIPES.md, as a solution to the "my viewer doesn't refresh after building"
problem?

On Thu, Jun 11, 2015 at 9:44 AM mindrunner <notifications@github.com> wrote:

> In build
> <https://github.com/shiblon/latex-makefile/pull/10#discussion_r32221238>:
>
> > @@ -1,4 +1,4 @@
> > -#!/usr/bin/env python
> > +#!/usr/bin/env python2.7
>
> yep, python2 is existing. should be the best solution here. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shiblon/latex-makefile/pull/10/files#r32221238>.
>